staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -292,21 +261,21 @@ func (s *APIAggregator) AddAPIService(apiService *apiregistration.APIService) {
 		serviceResolver: s.serviceResolver,
 	}
 	proxyHandler.updateAPIService(apiService)
+	if err := s.openAPIAggregator.loadApiServiceSpec(proxyHandler, apiService); err != nil {
+		return err
```

Comment:
what are the consequences of such an error? If I see it correctly, we keep the old handler installed? Will the aggregator still be able to serve the old openapi spec?